CONTRIBUTING.md

```diff
@@ -11,6 +11,8 @@ black .
 ```
 Note that it's important that `isort` is run first, because it doesn't produce a format that's consistent with `black`. 
 
+After pushing your changes, remember to pull from the remote repository. Automatic formatting changes might be made immediately after you push, so its important to ensure your local repository is up to date with the remote repository. 
```

Comment:
Minor comment: you say the changes are made immediately after you push. I expect it takes a few seconds to run. It might be worth saying that so someone doesn't run `git push` and then half a second later `git pull`. If I were a new contributor, I might not realize how things work and make this mistake. 

Unless there's something I'm missing and you actually could do `git push; git pull` with no delay.

Reply:
Good point usually the workflow takes a few seconds to run so I will update that accordingly.